serverlessworkflow/modules/ROOT/pages/eventing/working-with-callbacks.adoc

```diff
@@ -37,9 +54,11 @@ The first step to be done is to declare the event types being used by the workfl
   ]
 ----
 
-Then it declares a Callback state that will publish an event of type `resume` and wait for an event of type `wait`. The Cloud Event being published contains a `move` data field. The event being received is expected to have a `result` data field, which according to the link:{spec_doc_url}#event-data-filters[eventDataFilter], will be added to the workflow model as `move` field. 
+After that, a Callback state is declared. The Callback state publishes a `resume` type CloudEvent and waits for a CloudEvent with `wait` type. The published CloudEvent contains a `move` data field, and the CloudEvent that is received is expected to contain a `result` data field. According to the link:{spec_doc_url}#event-data-filters[eventDataFilter], the `result` data field is added to the workflow model as a `move` field.
```

Comment:
I do not understand why this is converted to passive voice. And other sentences in passive voice are converted to a non passive voice. I start to think we are changing by the sake of it. Can you explain the criteria for using passive?

Reply:
@fjtirado Please see the following screenshot from the IBM style guide explaining the criteria of using passive voice that we try to align with: 

![image](https://user-images.githubusercontent.com/59050394/180424748-d3ff8011-9731-4521-bece-bde78b8a7f76.png)

Reply:
so, which criteria applies to this change from active to passive?

Reply:
The following criteria is applied to this change:

- The information is clearer in passive voice. 
- Write in the simple present tense as much as possible.
- Using past or future tense only when you cannot use present tense or it does not make sense to use the present tense.

![image](https://user-images.githubusercontent.com/59050394/180429688-4d21b65a-aec4-401e-baa3-6cee43cd586a.png)

Reply:
So why not just move to present and do not change to pasive. 
Clearer is pretty subjetive concept.

Reply:
@fjtirado Is the modified sentence changing the meaning?

Reply:
no, in this particulars case, the meaning is the same, I was just curious about it being moved to passive.